.env.dev

```
@@ -0,0 +1,10 @@
+#/-------------------[DOTENV_PUBLIC_KEY]--------------------/
+#/            public-key encryption for .env files          /
+#/       [how it works](https://dotenvx.com/encryption)     /
+#/----------------------------------------------------------/
+DOTENV_PUBLIC_KEY_DEV="02c3930f5bea0a7548f755a4af0c8bdd4a228dd0789c38958c65e2a059a4779228"
+
+# .env.dev
+.env.dev
```

dotenv-linter (4.0.0)

[warning] [IncorrectDelimiter] The .env.dev key has incorrect delimiter

(IncorrectDelimiter)